source/Camera/InfoUtilOther/WarmupSample/WarmupSample.cs

```diff
@@ -0,0 +1,82 @@
+﻿
+/*
+This example shows a basic warm up method for a Zivid camera.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Globalization;
+using System.Threading;
+using Duration = Zivid.NET.Duration;
+
+
+
+namespace WarmupSample
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			try
+			{
+
+				var warmupTime = TimeSpan.FromMinutes(10);
+				var captureCycle = TimeSpan.FromSeconds(5);
+				var maxCaptureTime = Duration.FromMilliseconds(1000);
+
+				var zivid = new Zivid.NET.Application();
+
+				Console.WriteLine("Connecting to camera");
+				var camera = zivid.ConnectCamera();
+
+				Console.WriteLine("Getting suggested parameters");
+
```

Comment:
"Getting camera settings" to be consistent with https://github.com/zivid/zivid-python-samples/blob/master/source/camera/info_util_other/warm_up.py and https://github.com/zivid/zivid-cpp-samples/blob/master/source/Camera/InfoUtilOther/Warmup/Warmup.cpp

and remove blank line below